bin/check-bldr-toml.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# pre-commit hook:
+# Check for plans that don't have an entry in our .bldr.toml
+#
+# Authors:
+# - Steven Danna <steve@chef.io>
+
+set -eu
+requested=($@)
+
+check_bldr_toml_for() {
+  plan_path=$1
+  # Alternatively, we could do
+  #     plan_name=$(dirname $plan_path)
+  # if we think directory names will always match plan names.
```

Comment:
This is indeed a requirement per [CONTRIBUTING.md](https://github.com/habitat-sh/core-plans/blob/master/CONTRIBUTING.md#package-name-conventions)

Reply:
Per @rsertelon this is an assumption we should make - it will also help catch the times when that's not true, and let us correct those occurences.

Reply:
👍 Sourcing the plan makes me nervous as well so I was hoping his was the response. :D